torch/csrc/generic/methods/TensorMath.cwrap

```diff
@@ -1299,8 +1299,10 @@
   with_stateless: True
   return: accreal
   arguments:
-    - THTensor* self
-    - THTensor* tensor
+    - arg: THTensor* self
+      checkdim: 1
+    - arg: THTensor* tensor
+      checkdim: 1
```

Comment:
how about `assert_ndim: 1`? `checkdim` doesn't really tell you what kind of check is it